.travis.yml

```diff
@@ -175,6 +184,5 @@ matrix:
         - package='libpcap'; [[ $(brew ls --versions ${package}) ]] && { brew outdated ${package} || brew upgrade ${package}; } || brew install ${package}
         - package='xz'; [[ $(brew ls --versions ${package}) ]] && { brew outdated ${package} || brew upgrade ${package}; } || brew install ${package}
         - package='libssh2'; [[ $(brew ls --versions ${package}) ]] && { brew outdated ${package} || brew upgrade ${package}; } || brew install ${package}
-
-script:
-  - ./bootstrap.sh -e -t && cd build  && make -j2 VERBOSE=1 && sudo make test ARGS="-j2 --output-on-failure" && make linter && make apache-rat
```

Comment:
I don't see apache-rat anywhere. I am not sure if we have ever really used it, just want to make sure that we won't miss it (I won't).

Reply:
We do not and it caused many more issues than it solved. We should validate that before release.